terraform/modules/aws-dns-records/variables.tf

```diff
@@ -0,0 +1,53 @@
+variable "environment" {
+  type        = string
+  description = "name of the environment as a scope for the created resources (default: 'dev'; example: 'prod', 'staging')"
+  default     = "dev"
+}
+
+variable "zone_fqdn" {
+  type        = string
+  description = "FQDN of the DNS zone root (required; example: example.com; will append: '.')"
+}
+
+variable "domain" {
+  type        = string
+  description = "name of the sub-tree all given subdomains are append to (defaults to $environment; example: $subdomains[0].$domain.$zone_fqdn)"
+  default     = null
+}
+
+variable "subdomains" {
+  type        = list(string)
+  description = "list of sub-domains that will be registered under the given root domain"
+  default = [
+    "https",
+    "ssl",
```

Comment:
Please adhere to the names in https://docs.wire.com/how-to/install/helm.html#how-to-set-up-dns-records

Reply:
Would you be open to rather adjust the docs, since the docs also refer to the use of https://github.com/wireapp/wire-server-deploy/blob/develop/ansible/roles/minio-static-files/defaults/main.yml#L9-L14, where the module actually adheres.

Reply:
https://github.com/wireapp/wire-server-deploy/blob/develop/ansible/roles/minio-static-files/defaults/main.yml#L9-L14 is incorrect and should be changed.

Currently the docs, and all the DNS names used there, as well as in all the demo/prod/default values should be used for the purpose of this PR.

If you want to change names, please 1) open an issue with a motivation, and 2) make a PR that ensures consistency here across all DNS names in all of our configuration files - but make this a separate PR, not a side effect of a PR introducing a new terraform module. Thanks.